src/snarky.d.ts

```diff
@@ -167,24 +447,24 @@ export class Bool {
 
   static count(x: Bool | boolean[]): Field;
 
-  static sizeInFieldElements(): number;
-  static toFieldElements(x: Bool | boolean): Field[];
-  static ofFieldElements(fields: Field[]): Bool;
+  static sizeInFields(): number;
+  static toFields(x: Bool | boolean): Field[];
+  static ofFields(fields: Field[]): Bool;
 
   static toJSON(x: Bool): JSONValue;
   static fromJSON(x: JSONValue): Bool | null;
 }
 
 export interface AsFieldElements<T> {
-  toFieldElements(x: T): Field[];
-  ofFieldElements(x: Field[]): T;
-  sizeInFieldElements(): number;
+  toFields(x: T): Field[];
+  ofFields(x: Field[]): T;
+  sizeInFields(): number;
 }
 
 export interface CircuitMain<W, P> {
   snarkyWitnessTyp: AsFieldElements<W>;
   snarkyPublicTyp: AsFieldElements<P>;
-  snarkyMain: (W, P) => void;
+  snarkyMain: (w: W, p: P) => void;
 }
 
 export class Circuit {
```

Comment:
Since Circuit has only static methods, I think it shouldn't be a class. It should just be a module/namespace - object with functions on it